shard.yml

```diff
@@ -20,7 +20,7 @@ dependencies:
     branch: master
   kemal-session:
     github: kemalcr/kemal-session
-    commit: a16f164d3dcfc6e720d566f9085285abdd5b7825
+    version: ~> 0.10.0
```

Comment:
This should be a separate commit or PR, right?

Reply:
Can be, ive reverted it.

Reply:
Add it as a separate commit here so the build passes and I can merge.

Reply:
It wont pass, because even the newest version of `kemalcr/kemal-session` is broken on 0.34.0. We cant update the logger before 0.34.0 (because `Log` doesnt exist yet). So we need to wait for a 0.34.0 version of `kemalcr/kemal-session`. I will try to make a PR there as well.